packages/tools/kda-cli/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "kda-cli",
+  "version": "0.0.0",
+  "license": "MIT",
+  "bin": "dist/cli.js",
+  "type": "module",
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "test": "prettier --check . && xo && ava",
+    "kda": "node dist/cli.js"
+  },
```

Comment:
`_phase` scripts are missing (see `pactjs-cli` for guidance, I think that one has most of the repo stuff covered)
`lint-staged` is missing
The config for `lint-staged` can be found in another package as well